app/uk/gov/hmrc/eusubsidycompliancefrontend/controllers/BusinessEntityController.scala

```diff
@@ -52,6 +60,9 @@ class BusinessEntityController @Inject()(
 
   import escActionBuilders._
   val eoriPrefix = "GB"
+  val ADD_MEMBER_EMAIL_BE = "addMemberEmailToBE"
```

Comment:
Scala convention for constants is camelcase starting with a capital letter e.g. `AddMemberEmailBusinessEntity`